### DISCUSSION TITLE
Feature Request: Display renamed/removed folders more conveniently in PR review page

### DISCUSSION THREAD

**Discussion by LinqLover** [community/community#7049] [Pull Requests]
Whenever I rename/move/remove an entire folder in a commit/PR, the change list displays every file individually. In many situations, the diff would be easier to read if there would just be a single entry for the entire folder with the old and new name.

**Comment by fromsomnia** [comment]
Perhaps a directory view for files changed where you could check off (or uncheck) an entire folder/subtree when doing a review and have the ability to minimize/expand folders with an appropriate default state (minimized for directories containing tens/hundreds/thousands of files. This would handle all use cases equally well. Pretty please.
